### PR TITLE
Move more logic into API from UI for Notebook Clusters

### PR DESCRIPTION
**Pull request by blrubenstein**
This is a followup to the PR for initial notebook cluster creation, to move more of the logic into the API.

### CODE REVIEW

File: api/src/main/java/org/pmiops/workbench/api/ClusterController.java [hunk 1]
```diff
@@ -35,26 +37,25 @@
...
 
-  private org.pmiops.workbench.notebooks.model.ClusterRequest toFirecloudClusterRequest(ClusterRequest allOfUsClusterRequest) {
+  private org.pmiops.workbench.notebooks.model.ClusterRequest createFirecloudClusterRequest() {
     org.pmiops.workbench.notebooks.model.ClusterRequest firecloudClusterRequest = new org.pmiops.workbench.notebooks.model.ClusterRequest();
-    firecloudClusterRequest.setBucketPath(allOfUsClusterRequest.getBucketPath());
-    firecloudClusterRequest.setServiceAccount(allOfUsClusterRequest.getServiceAccount());
-    firecloudClusterRequest.setLabels(allOfUsClusterRequest.getLabels());
-    firecloudClusterRequest.setJupyterExtensionUri(allOfUsClusterRequest.getJupyterExtensionUri());
+    // TODO: Use real paths and accounts.
+    firecloudClusterRequest.setBucketPath("");
```
Comment (danqrodney) [inline api/src/main/java/org/pmiops/workbench/api/ClusterController.java]: Does this actually work? Where do the notebook files go?

File: api/src/main/java/org/pmiops/workbench/api/ClusterController.java [hunk 2]
```diff
@@ -35,26 +37,25 @@
...
+    firecloudClusterRequest.setServiceAccount("");
```
Comment (danqrodney) [inline api/src/main/java/org/pmiops/workbench/api/ClusterController.java]: I think based on this:

https://docs.google.com/document/d/1lcUUX_O2vA9fWwflnhaTpvAlaVqMb47olz4lS1UKwc8/edit?ts=59d5394c

in future we won't have to pass this in?

File: api/src/main/java/org/pmiops/workbench/api/ClusterController.java (hunk 1 above)
Comment (blrubenstein) [reply api/src/main/java/org/pmiops/workbench/api/ClusterController.java]: Currently they just get saved in the cluster. (Which isn't a google bucket.) There's currently no support on their side for saving files off to google buckets.

File: api/src/main/java/org/pmiops/workbench/api/ClusterController.java (hunk 2 above)
Comment (blrubenstein) [reply api/src/main/java/org/pmiops/workbench/api/ClusterController.java]: That may very well be correct. Current swagger docs say that we need to, but I'll keep a close eye on it.

**Pull request merged**